Testing/xASL_test_CompareReference.m

```diff
@@ -0,0 +1,223 @@
+function [result] = xASL_test_CompareReference(pathReference, pathResults)
```

Comment:
Is this copy pasted without changes from xASL_test_ExploreASL - then these should all be removed from there to avoid having two copies of the same code.

Reply:
Some changes were made, but I'll change xASL_test_ExploreASL to call this instead of something else.